src/contact.md

```diff
@@ -1,33 +1,45 @@
 # Contact
 
-**Seminar Location:**
+## **Seminar Locations:**
 
-[Cullen Family Lecture Theatre](https://conferencecentre.providencehealthcare.org/facilities/cullen-family-lecture-theatre) (Room 1477)<br>
-Providence Building, Level 1<br>
-St. Paul's Hospital<br>
-[1081 Burrard Street, Vancouver, BC](https://goo.gl/maps/uCFbWCXcVrLmMnch7)<br>
-V6Z 1Y6, Canada
+The speaker will be present in-person at one location while the other location receives live video-streaming and Q&A. Both locations will have a post lecture social with free food and drink.
 
-**Email:** wwhsiao \_AT_ sfu \_DOT_ ca
+**SFU's Big Data Hub**<br>
+[The Presentation Studio (ASB 10900)](https://www.sfu.ca/big-data/services/meeting-event-spaces.html)<br>
+[Applied Sciences Bldg, Burnaby, BC](https://maps.app.goo.gl/TtQruxPNSzihmb7k9)<br>
+V5A 0A7, Canada
 
-**Mailing Address:**
+**UBC's Michael Smith Laboratories**<br>
+[Room 102: Lecture Theatre](https://www.msl.ubc.ca/room/room-102-lecture-theatre/)<br>
```

Comment:
The links are kind of an odd choice here. I find it confusing that the second line but not the 3rd is a google maps link. Maybe better to just not make the text the link and put it in parenthesis "(google maps)" instead?

Reply:
Sure! I was just following the pattern for how it was set-up previously. Do you mind pushing that modification?